pages/sdk/bako-standalone/vault.mdx

```diff
@@ -13,6 +13,26 @@ const params: IPayloadVault = {
 }
 
 const vault = await Vault.create(params)
-
-console.log(vault.address.toString())
 ```
+It is also possible to create a vault with a specific [predicate contract version](../version).
+
+```ts
+import { IPayloadVault, Vault, BakoSafe } from 'bakosafe'
+import { Address } from 'fuels'
+
+// code is a unique string that identifies your predicate contract
+const predicateVersion = await Vault.BakoSafeGetPredicateVersionByCode(code)
```

Comment:
add an const to code of safe version,
with a code of this required configurables